test_game.py

```diff
@@ -0,0 +1,150 @@
+"""The test game."""
+
+
+PAYOFF_TABLE = [[2, 3],  # payoff_table[my_move][their_move]
+                [-3, 1]
+               ]
+PLAYER_NUMBER = 2
+ROUND_NUMBER = 5
+
+
+class MoveInfo:
+    """@brief A class containing all information about a player's turn.
```

Comment:
For the comment: "A class containing all information about" can be removed. "A player's turn." seems neater.